### ISSUE TITLE
Add world container folder to settings.json

### ISSUE THREAD

**Issue by K3rn31pan1c** [VolmitSoftware/Iris#817]
### Adding, Removing, or Changing

Adding

### Type of Modification

I want to put iris with the rest of my worlds 

### What are you trying to modify

How would I implement this? No idea...but I think it would be nice to have my Iris world with my other worlds in the folder I set in bukkit.yml. Seems like something I'd find in settings.json. My worlds have gotten large and I keep them on a separate SSD.

### Alternatives

If this could not be implemented I would ...Flip hot flap jacks with my bare hands until somebody cared...

**Comment by svdgoor** [comment]
Are you saying you have your worlds on another drive? Cool.
I think we get your worlds from the bukkit API so having them on another drive should just work. If it does not, can you send a log file here, as well as your bukkit.yml + where your world is on your system?
Would help us implement this :)

**Comment by K3rn31pan1c** [comment]
Well I did have my worlds on a separate drive but recently moved them back to my main drive because of weird I/O errors XD

So my folder structure is:
purpur directory : /home/xavier/PURPUR_SERVER/`purpur
worlds directory: /home/xavier/PURPUR_SERVER/worlds

My main world is set in server.properties using level-name=Namarith is in the worlds directory

When I start purpur and Iris is loaded, it creates a folder called Namarith in the purpur directory. /home/xavier/PURPUR_SERVER/`purpur/Namarith with a datapacks folder and Iris always says it found 0 biomes and I'm not sure why it does this.

I would like to be able to tell Iris to put it in my worlds directory

Bukkit.yml https://pastebin.com/SDMYEXQv
log            https://pastebin.com/Fa1QT9GU

**Comment by NextdoorPsycho** [comment]
on a scale from 1 to  "not my job" 
this is "not my job"
If you want a world on a second partition you will need to make a plugin to do that. Iris does not  manage worlds (with the exception of when bukkit misses us *cough*) but if you can get a plugin to point iris to where to make and manage a world, it can be in 50 folders, iris dont care. this is not something we intend to do, this is outside our scope for just a dimension engine. 

TLDR: get a plugin that lets you do this, and iris will just work with it

**Comment by K3rn31pan1c** [comment]
I just don't get why Iris doesn't respect the world-container: option in bukkit.yml. :( ty for your time
